weasyl/submission.py

```diff
@@ -217,8 +230,11 @@ def create_visual(userid, submission,
         submitid, 'submission', submit_media_item)
     orm.SubmissionMediaLink.make_or_replace_link(
         submitid, 'cover', cover_media_item)
-    orm.SubmissionMediaLink.make_or_replace_link(
-        submitid, 'thumbnail-generated', thumb_generated_media_item)
+
+    for thumb_generated_media_item in thumb_generated_media_items:
+        orm.SubmissionMediaLink.make_or_replace_link(
```

Comment:
If `thumb_generated_media_items` has multiple elements, is this just going to replace the same link repeatedly?